### PR TITLE
Construct storage from config.type

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

File: src/uptane/initialize.cc [hunk 1]
```diff
@@ -108,7 +108,7 @@ void Repository::setEcuKeysMembers(const std::string& primary_public, const std:
   key_source = source;
 }
 
-// Postcondition: (public, private) is in the storage. It should not be stored until secondaries are provisioned
+// Postcondition: (public, private) is in the storage-> It should not be stored until secondaries are provisioned
```
Comment (patrickvacek) [inline src/uptane/initialize.cc]: This is an amusing typo. :)

**Comment by patrickvacek** [comment]
This looks good, but I think uptane_ci_test also needs an update.

**Comment by patrickvacek** [comment]
I've tested with an actual device, and it works! Very cool. However, aktualizr-info does not. It needs to be able to read from the database, too.

**Comment by patrickvacek** [comment]
Actually, I can't send an update to the device. I'm seeing `Error of pulling image: Server returned HTTP 503`.

File: src/uptane/initialize.cc (hunk 1 above)
Comment (patriotyk) [reply src/uptane/initialize.cc]: That's was an  replace all  ))))))

**Comment by patriotyk** [comment]
Fixed uptane_ci_test and aktualizr-info

**Pull request merged**